src/Objs/Job/JobEditingConfig.js

```diff
@@ -90,4 +90,31 @@ Scrivito.provideEditingConfig("Job", {
     ...metadataInitialContent,
     body: [new SectionWidget({})],
   },
+  validations: [
+    [
+      "title",
+
+      title => {
+        if (!title) {
+          return {
+            message: "Job title must be set.",
+            severity: "error",
+          };
+        }
+      },
+    ],
+    [
+      "validThrough",
+
+      (validThrough, { obj }) => {
+        const datePosted = obj.get("datePosted");
+        if (datePosted && validThrough && datePosted >= validThrough) {
+          return {
+            message: '"posted at" must be before "expires at".',
```

Comment:
Since this shows up as an error of expires at I would switch the message to:

`"Expires at" must be after "Posted at".`

Reply:
"The expiration date must be after the posting date."?